services/release-controller-test/admin_02_rbac.yaml

```diff
@@ -0,0 +1,183 @@
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: system:authenticated
+  roleRef:
+    kind: ClusterRole
+    apiGroup: rbac.authorization.k8s.io
+    name: system:image-puller
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1beta1
+  metadata:
+    name: user-viewer
+    namespace: release-controller-test-release
+  subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: system:authenticated
+  roleRef:
+    kind: ClusterRole
+    apiGroup: rbac.authorization.k8s.io
+    name: view
+
+# grant release-controller-test image modification in the release namespace
+- kind: Role
+  apiVersion: authorization.openshift.io/v1
+  metadata:
+    name: release-controller-test-modify
+    namespace: release-controller-test-release
+  rules:
+  - apiGroups:
+    - ""
+    resources:
+    - secrets
+    resourceNames:
+    - release-upgrade-graph
+    verbs:
+    - get
+    - update
+    - patch
+  - apiGroups:
+    - image.openshift.io
+    resources:
+    - imagestreams
+    - imagestreamtags
+    verbs:
+    - get
+    - list
+    - watch
+    - create
+    - delete
+    - update
+    - patch
+  - apiGroups:
+    - ""
+    resources:
+    - events
+    verbs:
+    - create
+    - patch
+    - update
+- kind: Role
+  apiVersion: authorization.openshift.io/v1
+  metadata:
+    name: release-controller-test-import
+    namespace: release-controller-test-release
+  rules:
+  - apiGroups:
+    - image.openshift.io
+    resources:
+    - imagestreamimports
+    verbs:
+    - create
+- kind: Role
+  apiVersion: authorization.openshift.io/v1
+  metadata:
+    name: release-controller-test-prowjob
+    namespace: release-controller-test-job
+  rules:
+  - apiGroups:
+    - prow.k8s.io
+    resources:
+    - prowjobs
+    verbs:
+    - get
+    - list
+    - watch
+    - create
+    - delete
+    - update
+    - patch
+
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: release-controller-test-binding
+    namespace: release-controller-test-release
+  roleRef:
+    kind: Role
+    name: release-controller-test-modify
+  subjects:
+  - kind: ServiceAccount
+    namespace: release-controller-test
+    name: release-controller-test
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: release-controller-test-binding-view
+    namespace: release-controller-test-release
+  roleRef:
+    kind: ClusterRole
+    name: view
+  subjects:
+  - kind: ServiceAccount
+    namespace: release-controller-test
+    name: release-controller-test
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: release-controller-test-binding-prowjob
+    namespace: release-controller-test-job
+  roleRef:
+    kind: Role
+    name: release-controller-test-prowjob
+  subjects:
+  - kind: ServiceAccount
+    namespace: release-controller-test
+    name: release-controller-test
+
+# grant release-controller-test edit on its work namespace
```

Comment:
nit: `edit` is a nice thing for users, it's better to correctly scope verbs and nouns for service accounts